lib/modules/ai_bot/jobs/regular/create_ai_reply.rb

```diff
@@ -6,10 +6,25 @@ class CreateAiReply < ::Jobs::Base
 
     def execute(args)
       return unless bot_user = User.find_by(id: args[:bot_user_id])
-      return unless bot = DiscourseAi::AiBot::Bot.as(bot_user)
       return unless post = Post.includes(:topic).find_by(id: args[:post_id])
 
-      bot.reply_to(post)
+      kwargs = {}
+      kwargs[:user] = post.user
+      if persona_id = post.topic.custom_fields["ai_persona_id"]
+        persona_id = persona_id.to_i
+        kwargs[:persona_id] = persona_id
```

Comment:
```suggestion
        kwargs[:persona_id] = persona_id.to_i
```

Could just do this and get rid of previous row